docs/jwst/references_general/references_general.rst

```diff
@@ -162,7 +162,7 @@ documentation on each reference file.
 +--------------------------------------------------+---------------------------------------------+
 | Reference File Type (REFTYPE)                    | Pipeline Step                               |
 +==================================================+=============================================+
-| :ref:`ABVEGA_OFFSET <abvega_offset_reffile>`     | :ref:`source_catalog <source_catalog_step>` |
+| :ref:`ABVEGAOFFSET <abvegaoffset_reffile>`      | :ref:`source_catalog <source_catalog_step>` |
```

Comment:
Not sure if sphinx is going to care about the misalignment of the cell border markers "|" between this line and surrounding lines. Might need to add in some extra spaces to account for the shortening of the name.

Reply:
Fixed, thanks!